framework/bootstrap/http.go

```diff
@@ -14,17 +15,20 @@ import (
 
 // PathHandler for module using
 type PathHandler struct {
-	mux *http.ServeMux
+	mux     *http.ServeMux
+	mapping map[string]http.Handler
 }
 
 func NewPathHandler() *PathHandler {
 	return &PathHandler{
-		mux: http.NewServeMux(),
+		mux:     http.NewServeMux(),
+		mapping: make(map[string]http.Handler),
 	}
 }
 
 func (ph PathHandler) Handle(path string, handler http.Handler) {
 	ph.mux.Handle(path, handler)
+	ph.mapping[path] = handler
```

Comment:
we'd better use lock to protect it.

not quite sure that it'll not be called concurrently

Reply:
add RWMutex